build.ps1

```diff
@@ -353,11 +356,21 @@ try
             Write-Host -ForegroundColor Magenta "IMPORTANT: Using local packages."
         }
 
+        # Tests categories to include
+        $testCategory = "("
+        $testCategory += "TestCategory=E2E"
+        if (-not(IsWindows)) 
```

Comment:
Can you document here (as a code comment) why we don't run on Windows?

Reply:
updated on latest commit